### PR TITLE
feature/employee sees list of bought rewards

### PR DESCRIPTION
**Pull request by kajawitek**
Hi Nerds! 🤓

I added new features to our app:

1. Employee has list of bought rewards view.
2. Changing reward price by admin does not affect the price displayed in the list of bought rewards
3. Specs check that employee can see list of bought rewards and that changing reward price does not affect the price in list of bought rewards.

### CODE REVIEW

File: app/views/orders/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <th>Price</th>
+
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @orders.each do |order| %>
+      <tr>
+      <% if current_employee == order.employee %>
```
Comment (ciejj) [inline app/views/orders/index.html.erb:20]: this is quite a lot of comparisons, what if there are 10_000_000 of orders in the db, but only 1 belongs to current user? Consider filtering out only orders belonging to the current user in controller - something like `current_user.orders`

File: spec/features/order_spec.rb [hunk 2]
```diff
@@ -3,15 +3,37 @@
...
 
-  it 'shows a lower number of points after buying a reward' do
+  it 'tests orders' do
+    # showing a lower number of points after buying a reward
     click_link 'Buy'
     expect(page).to have_content 'Order was successfully created'
     expect(page).to have_content 'Your points: 1'
+
+    # seeing bought rewards list by employees
+    visit orders_path
```
Comment (ciejj) [inline spec/features/order_spec.rb:21]: consider changing it to `click_link`

**Review by ciejj — APPROVED** [review]
Good work! left couple of ideas for improvements :)

File: app/views/orders/index.html.erb (hunk 1 above)
Comment (kajawitek) [reply app/views/orders/index.html.erb]: done :)

File: app/views/pages/_header.html.erb [hunk 3]
```diff
@@ -20,6 +20,9 @@
             <li class="nav-item">
               <%= link_to "Rewards", rewards_path, class:"nav-link" %>
             </li>
+            <li class="nav-item">
+              <%= link_to "My rewards", orders_path, class:"nav-link" %>
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply app/views/pages/_header.html.erb]: done :)

File: spec/features/order_spec.rb (hunk 2 above)
Comment (kajawitek) [reply spec/features/order_spec.rb]: done :)

**Comment by kajawitek** [comment]
Thank you :)

**Pull request merged**